src/cli/commands/status.js

```diff
@@ -35,25 +36,81 @@ exports.handler = (opts) => {
-          } else if (status.metadata.debug && (status.metadata.debug === 'true' || status.metadata.debug === true)) {
-            utils.writer.write(', with ' + 'DEBUG'.yellow + ' enabled on port ' + (opts.debugPort || 5858));
+        let time = Math.floor((Date.now() - config.started) / (1000));
+        if (time > (60 * 60)) {
+          time = `${Math.floor(time / (60 * 60))} hour(s)`;
+        } else if (time > 60) {
+          time = `${Math.floor(time / 60)} minute(s)`;
+        } else {
+          time = `${Math.floor(time)} seconds`;
+        }
+
+        table.push(['Status'.white, 'RUNNING'.green]);
+        table.push(['Uptime'.white, time]);
+        table.push(['Process ID'.white, config.pid]);
+        table.push(['Rest Service'.white, `http://${config.restHost}:${config.restPort}/`]);
+        table.push(['gRPC Service'.white, `http://${config.grpcHost}:${config.grpcPort}/`]);
+        if (config.inspect || config.debug) {
+          table.push(['Debugger'.white, 'ACTIVE'.green]);
+          if (config.inspect) {
+            table.push(['Debugger Port'.white, `${config.inspectPort}`.white]);
+          } else {
+            table.push(['Debugger Port'.white, `${config.debugPort}`.white]);
           }
+        } else if (config.debug) {
+          table.push(['Debugger'.white, 'INACTIVE'.yellow]);
+        }
+        table.push(['Log file'.white, config.logFile]);
+        table.push(['Project ID'.white, config.projectId]);
+        table.push(['Region'.white, config.region]);
+        table.push(['Storage Mode'.white, config.storage]);
+
+        if (config.mocks) {
+          table.push(['Mocks'.white, 'LOADED'.green]);
+        } else {
+          table.push(['Mocks'.white, 'NOT LOADED'.yellow]);
+        }
+
+        table.push([{ colSpan: 2, content: 'Supervisor'.cyan }]);
+
+        if (config.runSupervisor) {
+          table.push(['Status'.white, 'RUNNING'.green]);
+          table.push(['Isolation Mode'.white, config.isolation]);
+        } else {
+          table.push(['Status'.white, 'DETACHED'.yellow]);
         }
 
-        utils.writer.write('\n');
+        table.push(['Trigger URL'.white, `http://${config.supervisorHost}:${config.supervisorPort}/${config.projectId}/${config.region}/FUNCTION_NAME`]);
       } else {
-        utils.writer.write(' STOPPED\n'.yellow);
+        let time;
+        if (config.stopped) {
+          time = Math.floor((Date.now() - config.stopped) / (1000));
+          if (time > (60 * 60)) {
+            time = `${Math.floor(time / (60 * 60))} hour(s)`;
+          } else if (time > 60) {
+            time = `${Math.floor(time / 60)} minute(s)`;
+          } else {
+            time = `${Math.floor(time)} second(s)`;
+          }
+        }
+
+        table.push(['Status'.white, 'STOPPED'.yellow]);
+        if (time) {
+          table.push(['Last up'.white, `${time.yellow} ago`]);
```

Comment:
This results in 'ago' being "default console text color", and the time being in yellow.. which looks a little odd to me